src/timestamp.ts

```diff
@@ -1,20 +1,10 @@
+import { BSONValue } from './bson_value';
 import { BSONError } from './error';
 import type { Int32 } from './int_32';
 import { Long } from './long';
 
 /** @public */
 export type TimestampOverrides = '_bsontype' | 'toExtendedJSON' | 'fromExtendedJSON' | 'inspect';
```

Comment:
delete this

Reply:
Was this meant to refer to the `TimeStampOverrides` type?